pkg/model/postgresql_backup.go

```diff
@@ -0,0 +1,176 @@
+package model
+
+import (
+	"github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	v13 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func PostgresqlBackup(cr *v1alpha1.KeycloakBackup) *v13.Job {
+	return &v13.Job{
+		ObjectMeta: v12.ObjectMeta{
+			Name:      cr.Name,
```

Comment:
Have you tested creating multiple local backups, it seems like the same name is used for each which would end up meaning only one would ever be created. This is the case with the PVC also.

Reply:
Good point. I abstracted `Containers[]`, which is the biggest part of the payload. Looks much better now.

Reply:
The `cr.name` is basically a name of the `KeycloakBackup` CR and needs to be unique across the namespace. Otherwise, you'll get an error like this:

```
Error from server (AlreadyExists): error when creating "deploy/examples/backup/OneTimeLocalBackup2.yaml": keycloakbackups.keycloak.org "example-keycloakbackup2" already exists
```

I double checked if this works correctly and created 2 `KeycloakBackup` objects. Everything seems to be working fine.

```
$ oc get keycloakBackups
NAME                      AGE
example-keycloakbackup    6m41s
example-keycloakbackup2   2m8s

$ oc get pvc
NAME                                      STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
keycloak-backup-example-keycloakbackup    Bound     pvc-14da8fe9-bff1-403b-b5c1-de54ed924a7b   1Gi        RWO            standard       6m43s
keycloak-backup-example-keycloakbackup2   Bound     pvc-357423eb-8aa0-44f9-873f-a50d99ab746f   1Gi        RWO            standard       2m10s
keycloak-postgresql-claim                 Bound     pvc-8b2266a6-5b0e-4272-a20a-68ce38e633fd   1Gi        RWO            standard       8d

$ oc get pods
NAME                                   READY     STATUS      RESTARTS   AGE
example-keycloakbackup-g78xt           0/1       Completed   0          6m45s
example-keycloakbackup2-ddqwp          0/1       Completed   0          2m12s
keycloak-0                             1/1       Running     0          6m55s
keycloak-postgresql-7dc64b9f59-cdf7c   1/1       Running     1          46h
```